tests/integrations/tso/client_test.go

```diff
@@ -214,7 +214,7 @@ func (suite *tsoClientTestSuite) TestRandomShutdown() {
 	wg.Add(1)
 	go func() {
 		defer wg.Done()
-		n := r.Intn(2) + 1
+		n := r.Intn(2) + 3
```

Comment:
why change it?

Reply:
This was changed during debugging a problem. Problem has been fixed but forgot to change it back. I'll change it back.

Reply:
changed back.

Reply:
changed to 3 again, please see the comment.